.github/workflows/promptflow-sdk-cli-test.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         pip install -r ${{ github.workspace }}/src/promptflow/dev_requirements.txt
         gci ./promptflow -Recurse | % {if ($_.Name.Contains('.whl')) {python -m pip install "$($_.FullName)[azure]"}} # Will remove azure deps later
         gci ./promptflow-tools -Recurse | % {if ($_.Name.Contains('.whl')) {python -m pip install $_.FullName}}
+        pip install -U flask ruamel.yaml
```

Comment:
remove this?

Reply:
remove ruamel.yaml and keep flask here since we need test for flask >=3.0.0